nix/buildPnpmPackage.nix

```diff
@@ -0,0 +1,61 @@
+# buildPnpmPackage.nix
```

Comment:
To make building PNPM packages easier.

@cor note the [`fileset`](https://nixos.org/manual/nixpkgs/stable/#sec-functions-library-fileset) usage which constructs a source derivation from only required files.